### PR TITLE
Feature/how to play

### PR DESCRIPTION
**Pull request by sawia**

### CODE REVIEW

File: src/howtoplay/howtoplay.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  color: #fff;
+  font-family: "Lato", sans-serif;
+}
+
+body {
+  background-image: url("/static/wave-background.png");
+  background-size: cover;
+}
+
+.main-container {
```
Comment (ruljin) [inline src/howtoplay/howtoplay.css]: Notation. Once you use underscore and another time dash again 😄

Comment (ruljin) [reply src/howtoplay/howtoplay.css]: I suggest using underscores in CSS.

File: src/howtoplay/howtoplay.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
+* {
```
Comment (ruljin) [inline src/howtoplay/howtoplay.css]: Reset styles should be moved to the global styles file.

File: src/howtoplay/howtoplay.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
```
Comment (ruljin) [inline src/howtoplay/howtoplay.css]: Should be moved to the global styles file, imported here from it under the variable.

File: src/howtoplay/howtoplay.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  color: #fff;
+  font-family: "Lato", sans-serif;
```
Comment (ruljin) [inline src/howtoplay/howtoplay.css]: I mentioned earlier (global styles and variables).

File: src/howtoplay/howtoplay.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+body {
+  background-image: url("/static/wave-background.png");
```
Comment (ruljin) [inline src/howtoplay/howtoplay.css]: All of you gonna use the same background image. Should be moved to the global styles file, imported here from it under the variable - should work for images in CSS also.

File: src/howtoplay/howtoplay.css [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  background-size: cover;
+}
+
+.main-container {
+  min-height: 100vh;
+  width: 65vw;
+  background-color: rgba(0, 0, 0, 0.45);
```
Comment (ruljin) [inline src/howtoplay/howtoplay.css]: Should be moved to the global styles file, imported here from it under the variable.

File: src/howtoplay/howtoplay.css [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.container__title {
+  font-size: 65px;
+  font-weight: 700;
```
Comment (ruljin) [inline src/howtoplay/howtoplay.css]: Should be moved to the global styles file, imported here from it under the variable.

File: src/howtoplay/howtoplay.css [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+  padding: 0 10px 10px 10px;
+  border-bottom: 5px solid #fff;
+  text-align: center;
+  margin-bottom: 5%;
+  margin-top: 5%;
+}
+
+.container__instruction {
+  width: 70%;
+  font-size: 24px;
```
Comment (ruljin) [inline src/howtoplay/howtoplay.css]: Should be moved to the global styles file, imported here from it under the variable.

File: src/howtoplay/howtoplay.css [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+.return {
+  align-self: flex-end;
+  margin-bottom: 5%;
+  margin-right: 5%;
+  font-size: 40px;
+  text-decoration: none;
+  transition: all .2s ease-in-out;
+}
+
+.return:hover {
```
Comment (ruljin) [inline src/howtoplay/howtoplay.css]: If it wasn't discussed with the team (or it is not mentioned on Figma) you should inform them about the behaviour of the return link on hover.

File: src/howtoplay/howtoplay.html [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Battleship - How to play?</title>
+    <link rel="stylesheet" href="howtoplay.css">
+</head>
+<body>
+    <div class="main-container">
```
Comment (ruljin) [inline src/howtoplay/howtoplay.html]: It is the main content of this page so it should be a `<main>` tag.

Comment (ruljin) [reply src/howtoplay/howtoplay.html]: And after changing it a class called `main` would be a good one also, solving two problems with it I mentioned 😄

File: src/howtoplay/howtoplay.scss [hunk 11]
```diff
@@ -0,0 +1,64 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
```
Comment (ruljin) [inline src/howtoplay/howtoplay.scss]: If you use `.scss` why do you need `.css` files 😄 ?

**Review by ruljin — CHANGES_REQUESTED** [review]

File: src/howtoplay/howtoplay.scss (hunk 11 above)
Comment (ruljin) [reply src/howtoplay/howtoplay.scss]: You can create a JS file called `howToPlay.js` for example and you can import in it `./howtoplay.scss`. After building the version with Webpack - it should solve the problem with styling itself. For tests, it is good to create a 'scss' file and use the loader to translate it to 'css' and check if the page looks good - but there is no need to push the CSS file later.

Comment (ruljin) [reply src/howtoplay/howtoplay.scss]: Ohh and name files using `camelCase` notation.

File: src/howtoplay/howtoplay.css (hunk 2 above)
Comment (JustD1d1t) [reply src/howtoplay/howtoplay.css]: moved to styles.scss

File: src/howtoplay/howtoplay.css (hunk 3 above)
Comment (JustD1d1t) [reply src/howtoplay/howtoplay.css]: same comment as before

File: src/howtoplay/howtoplay.css (hunk 4 above)
Comment (JustD1d1t) [reply src/howtoplay/howtoplay.css]: moved to variables

File: src/howtoplay/howtoplay.css (hunk 5 above)
Comment (JustD1d1t) [reply src/howtoplay/howtoplay.css]: moved

File: src/howtoplay/howtoplay.css (hunk 7 above)
Comment (JustD1d1t) [reply src/howtoplay/howtoplay.css]: Why should i write font-weight to variable? I can also use bold(700), regular(400) and probaly light(300) if i remember

File: src/howtoplay/howtoplay.css (hunk 8 above)
Comment (JustD1d1t) [reply src/howtoplay/howtoplay.css]: moved

File: src/howtoplay/howtoplay.css (hunk 9 above)
Comment (JustD1d1t) [reply src/howtoplay/howtoplay.css]: IMO i think we can add this hover. Anyway i will ask team for this

File: src/howtoplay/howtoplay.html (hunk 10 above)
Comment (JustD1d1t) [reply src/howtoplay/howtoplay.html]: I hope i know what are you talking about. Change tag for main and add class "main" there?

File: src/howtoplay/howtoplay.scss (hunk 11 above)
Comment (JustD1d1t) [reply src/howtoplay/howtoplay.scss]: Imported styles in howtoplay.js

File: src/howtoplay/howtoplay.html (hunk 10 above)
Comment (ruljin) [reply src/howtoplay/howtoplay.html]: Correct. Change the used tag to `<main class="main">`

**Pull request merged**